bintool/bintool.cpp

```diff
@@ -648,7 +648,7 @@ std::vector<uint8_t> get_lm_hash_data(elf_file *elf, block *new_block, bool clea
             if (data.size() != seg->physical_size()) {
                 fail(ERROR_INCOMPATIBLE, "Elf segment physical size (%" PRIx32 ") does not match data size in file (%zx)", seg->physical_size(), data.size());
             }
-            if (seg->physical_size() && seg->physical_address() < new_block->physical_addr) {
+            if (seg->physical_size()) {
```

Comment:
what was this change?

Reply:
https://github.com/raspberrypi/picotool/commit/011111d8dfe291d66531cd82a4704df8688b7f54 - it allows signing code in scratch